go.mod

```diff
@@ -1,4 +1,4 @@
-module fleet
```

Comment:
use the full path for the module name and the imports if we really want to this as a module anywhere. 


For example:
https://github.com/nats-io/nats.go/blob/master/examples/nats-pub/main.go

```
github.com/nats-io/nats.go
```

so in our case it will be 
```
module github.com/elastic/fleet-server
```

and the imports:
```
import "github.com/elastic/fleet-server/internal/pkg/policy"
```

When you start major versioning the module there will be a version in the path as well

```
import "github.com/elastic/fleet-server/v3/internal/pkg/policy"
```